iron-multi-selectable.html

```diff
@@ -12,7 +12,15 @@
 <link rel="import" href="iron-selectable.html">
 
 <script>
-  /** @polymerBehavior Polymer.IronMultiSelectableBehavior */
+  /**
+   * ### Notable breaking changes between 1.x and 2.x (hybrid):
```

Comment:
Huh, it seems like the README is meant to be updated with stuff from these files anyways.